docs/04-api-spec.md

```
@@ -1565,243 +1621,871 @@ GET /api/chat/rooms
-| --- | --- | --- | --- |
-| `cursor` | String | N | 이전 메시지 기준 커서 (messageId 인코딩) |
-| `limit` | Integer | N | 페이지 크기 (기본 50, 최대 100) |
+|----------|------|------|------|
+| `before` | Long | N | 이 메시지 ID 보다 과거의 것만 조회. 미지정 시 최신 메시지부터 |
+| `limit` | Integer | N | 페이지 크기 (기본 30, 최대 100) |
 
-#### Response Body
+Response Body
 
-```json
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)